### ISSUE TITLE
Feature request: Delay in motion detection

### ISSUE THREAD

**Issue by sambul13** [ccrisan/motioneye#192]
Motioneye is mostly used by private consumers, not businesses. Ordinary people tend to leave their homes and small offices at random times, so setting Motion Detection Schedule isn't always suitable. In addition, if such Schedule is set for night time, during daytime still a user may need on-demand motion detection. Its not convenient to change of switch off the schedule all the time.

Please add option allowing to start motion detection on-demand "from now till stopped" without affecting the Schedule settings. Also, a delay is often needed when starting detection on demand, and needs to be optional. For example, a user may want to start detection 5 min after leaving the premises, so his own departure is not detected and notified.

**Comment by ccrisan** [comment]
Makes sense. I might consider implementing such a feature when I'll get the courage to touch the motion detection control code again :)
